### ISSUE TITLE
Colors not coming on console after adding pino-inspector

### ISSUE THREAD

**Issue by 80avin** [pinojs/pino-inspector#2]
## Before
1. Colors were rendered to terminal.
2. Configuration:
```js
const logger = pino({
  customLevels: {
    clickStream: 35,
  },
  level: Config.logLevel,
  enabled: !process.env.LOG_DISABLED,
})
```
3. Typical console output: `INFO` and `connection to mongodb -> connected` was colored
```
[1626251020511] INFO  (981119 on avinash-snapwiz-pc): connection to mongodb -> connected 
```

## After:
1. No colors in output
2. Only change done is in configuration: 
```js
const logger = pino({
  customLevels: {
    clickStream: 35,
  },
  level: Config.logLevel,
  enabled: !process.env.LOG_DISABLED,
  prettifier: pinoInspector,
  prettyPrint: true,
})
```

**Comment by mcollina** [comment]
I'm not sure why this is happening! 
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by 80avin** [comment]
Hey !
I had figured this out.
We were piping the output of node output to pino-pretty which used to colorize it. Hence, I was getting pretty output with colors even without configuring it in code.

After I added pino-inspector, it added pino-pretty with empty configuration from the code itself and affected the output of the program.
Simple fix for me was to drop the cli piping and add required props in the code itself.

While ideally pino-inspector should have a config flag to not modify the output stream, the workaround is easy.
I might consider a PR after playing with it more and if I find any other issue, but for now it works.
